basetest.pm

```diff
@@ -298,17 +310,15 @@ sub runtest {
             $details->{text}  = $text_fn;
             $details->{title} = 'Failed';
             push @{$self->{details}}, $details;
+            $self->run_post_fail("test $name died");
         }
         else {
```

Comment:
Whole `else` block can be removed since `$self->{result}` is set to 'fail' and the same code is then executed in next `if`. So there is reason to keep it `if` :)